contentcuration/contentcuration/frontend/channelList/views/ChannelListIndex.vue

```diff
@@ -67,11 +67,7 @@
               </VCard>
             </VFlex>
           </VLayout>
-          <keep-alive>
-            <router-view
```

Comment:
We can still use the keep-alive here - what advantage was accrued by removing it?

Reply:
Oh that might be a mistake. I needed to remove `<keep-alive>` from some other places to prevent modals (that will become regular pages at one point) from surviving (otherwise they would overlap and caused unexpected behavior in some edge cases). I am going to revert this change and check if everything is working properly.

Reply:
So this was the problematic place actually. When I keep `keep-alive`, then `ChannelModal` and `ChannelDetailsModal` won't be destroyed on close (will be just inactive instead), and there are some buggy edge cases related to that. For example, this navigation sequence doesn't work properly: detail dialog -> edit dialog -> detail dialog (a wrong modal can be displayed). I assume that we can return `keep-alive` as soon as these two dialogs are converted to regular pages to keep `ChannelList` alive. However I am not sure if it's a good idea to use it on the root app page because it would become default behavior for all new pages. Would rather prefer more explicit usage.